solarforecastarbiter/metrics/calculator.py

```diff
@@ -154,6 +157,12 @@ def calculate_deterministic_metrics(processed_fx_obs, categories, metrics,
                     'observation': obs,
                     'reference': ref_fx}, axis=1)
 
+    # Force `groupby` to be consistent with `interval_label`, i.e., if
+    # `interval_label == ending`, then the last interval should be in the bin
+    closed = datamodel.CLOSED_MAPPING[processed_fx_obs.interval_label]
+    if closed == "ending":
```

Comment:
```suggestion
    if closed == "right":
```

Reply:
hm, actually, is there a reason we need to look up closed here? Would it be sufficient to test `processed_fx_obs.interval_label == "ending"`?

Reply:
You're right; I'll revise to just test `processed_fx_obs.interval_label == "ending"`.